.github/workflows/CodeAnalysisAndCI.yml

```diff
@@ -36,14 +36,19 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           flags: unittests # optional
           name: jacoco-report-uploader # optional
-          fail_ci_if_error: true # optional (default = false)
```

Comment:
Same comment as in https://github.com/gchq/gaffer-experimental/pull/209:
Does this need to go?

Reply:
Have responded in #209 let me know your thoughts please